core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -696,11 +675,17 @@ public final class State implements Journaled {
      * @param clock the clock used for computing sync times
      */
     State(MountInfo mountInfo, Clock clock) {
-      mMountTable = new HashMap<>(10);
-      mMountTable.put(MountTable.ROOT, mountInfo);
+      addMountInfo(mountInfo);
       mUfsSyncPathCache = new UfsSyncPathCache(clock);
     }
 
+    private void addMountInfo(MountInfo mountInfo) {
+      mMountTable.put(mountInfo.getAlluxioUri().getPath(), mountInfo);
+      mMountIdTable.put(mountInfo.getMountId(), mountInfo);
+      mUfsPaths.insert(mountInfo.getUfsUri().toString()).setValue(mountInfo);
```

Comment:
i'm thinking maybe a separate `.setValue()` in the API is a bit redundant? Would it be possible the trienode just has the correct value on creation? Does that value need to change (otherwise can be `final`)?